downstream/modules/eda/proc-eda-activate-webhook.adoc

```diff
@@ -0,0 +1,73 @@
+[id="proc-eda-activate-webhook"]
+
+= Activating webhook rulebooks
+
```

Comment:
As a general rule, I will use a gerund for a high level task as you have it here. However, for the subtasks within this chapter, I would change to an imperative verb. That said, there is some nuance with regard to gerunds and it's really up to you.